### ISSUE TITLE
arima postestimation example needs clarification

### ISSUE THREAD

**Issue by eduedix** [statsmodels/statsmodels#2776]
http://www.statsmodels.org/dev/examples/notebooks/generated/statespace_sarimax_stata.html
In ARIMA Postestimation: Example 1 - Dynamic Forecasting:

```
mod = sm.tsa.statespace.SARIMAX(endog.ix[:'1978-01-01'], exog=exog.ix[:'1978-01-01'], order=(1,0,1))
fit_res = mod.fit(disp=0)
mod = sm.tsa.statespace.SARIMAX(endog, exog=exog, order=(1,0,1))
res = mod.filter(np.array(fit_res.params))
predict_res = res.predict(full_results=True)

predict = predict_res.forecasts
cov = predict_res.forecasts_error_cov
```

`predict_res = res.predict(full_results=True)` should be changed to `predict_res = res.get_prediction()`

**Comment by eduedix** [comment]
I got it now, my fault. Though I think you could remove

```
predict_res = res.predict(full_results=True)
```
and just do

``` 
predict = res.forecasts
cov = res.forecasts_error_cov
```
?

**Comment by eduedix** [comment]
I am guessing things have moved from ```.predict()``` to ```.get_prediction()```.

**Comment by ChadFulton** [comment]
Thanks for the report! This issue was fixed in be4eb7c, but the dev documentation hasn't been updated in a while.
